XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,573 @@
+<pre>
+Title:       <b>On-Chain Credentials</b>
+Revision:    <b>3</b>
+
+Author:      <a href="mailto:mvadari@ripple.com">Mayukha Vadari</a>
+
+Affiliation: <a href="https://ripple.com">Ripple</a>
+</pre>
+
+# On-Chain Credentials
+
+## Abstract
+
+The XRPL DID (Digital Identifier) amendment ([XLS-40](https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0040d-decentralized-identity)) empowers users to manage their digital identities on the XRP Ledger. While this amendment adds support for on-chain identity management and simplifies off-chain credential usage, maximizing blockchain technology's full potential for credential handling requires on-chain solutions.
+
+This document proposes a design to bridge this gap. It outlines the issuance, storage, and verification of credentials directly on the XRP Ledger, while still supporting the privacy needs of users.
+
+On-chain credentialing can streamline various processes within the XRPL ecosystem. For example, financial institutions can issue credentials on the XRP Ledger, attesting to a user's identity and compliance. This eliminates the need for repeated [KYC (Know Your Customer)](https://en.wikipedia.org/wiki/Know_your_customer) checks across different platforms, fostering a smoother user experience. By enabling secure credential management, this proposal unlocks the potential for a new wave of trust-based applications within the XRPL ecosystem.
+
+## 1. Overview
+
+This design adds support for creating, accepting, and deleting credentials. It also extends the existing [Deposit Authorization](https://xrpl.org/docs/concepts/accounts/depositauth) feature, allowing accounts to not only whitelist specific accounts, but also whitelist accounts with specific credentials.
+
+This proposal only supports blocking the interactions that Deposit Authorization supports, such as direct payments. However, future proposals could add support for credential-gating in other parts of the XRPL, such as AMMs or lending pools.
+
+We propose:
+* Creating a `Credential` ledger object
+* Modifying the `DepositPreauth` ledger object
+* Modifying the `DepositPreauth` transaction
+* Creating a `Credential` ledger object
+* Creating a `CredentialCreate` transaction type
+* Creating a `CredentialAccept` transaction type
+* Creating a `CredentialDelete` transaction type
+
+This feature will require an amendment, tentatively titled `featureCredentialAuth`.
```

Comment:
Typically we don't include the `feature` prefix when naming amendments. (The prefix is used in the `rippled` codebase but typically not displayed to users nor used as part of the amendment's official name, i.e. the part that gets hashed to determine the ID or the name we use in docs.)

```suggestion
This feature will require an amendment, tentatively titled `CredentialAuth`.
```